build.gradle

```diff
@@ -276,6 +276,14 @@ dependencyManagement {
         // Versions prior to 24.1.1 vulnerable to CVE-2018-10237
         dependency 'com.google.guava:guava:24.1.1-jre'
 
+        // CVE-2020-5398
+        dependencySet(group: 'org.springframework.security', version: '5.2.1.RELEASE') {
+            entry 'spring-security-web'
+            entry 'spring-security-config'
+            entry 'spring-security-core'
```

Comment:
can this be set as " entry  'spring-security-*'  "  ? Not sure whether it works..